src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  &__field {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $field-gap;
+    padding: $field-padding;
+    width: calendarWidth(
+      $cell-size, $field-gap, $field-padding
+    );
+  }
+
+  &__cell {
+    text-align: center;
+    line-height: $cell-size;
+    width: $cell-size;
+    height: $cell-size;
+
+    background-color: #eee;
+    border: 1px solid #000;
+    transition-duration: 0.5s;
```

Comment:
```suggestion
    transition:  transform 0.5s, background-color 0.5s;
```

Reply:
indicate what you want to add transition to

Reply:
you're right, thanks a lot